.github/workflows/pypi-build-artifacts.yml

```
@@ -70,13 +70,15 @@ jobs:
           CIBW_ARCHS: "auto64"
           CIBW_PROJECT_REQUIRES_PYTHON: ">=3.10"
           CIBW_TEST_REQUIRES: "pytest==7.4.2 moto==5.0.1"
-          CIBW_TEST_COMMAND: "pytest {project}/tests/avro/test_decoder.py"
+          CIBW_TEST_COMMAND: "PYTHON_GIL=1 pytest {project}/tests/avro/test_decoder.py"
```

```suggestion
          CIBW_TEST_COMMAND: "pytest {project}/tests/avro/test_decoder.py"
```
do we need this? since we added `CIBW_SKIP: "cp3*t-*"` below